pyproject.toml

```diff
@@ -1,47 +1,51 @@
-[tool.poetry]
+[project]
 name = "aleph-alpha-client"
 version = "10.0.0"
 description = "python client to interact with Aleph Alpha api endpoints"
-authors = ["Aleph Alpha <support@aleph-alpha.com>"]
-license = "MIT"
+authors = [{ name = "Aleph Alpha", email = "support@aleph-alpha.com" }]
+requires-python = ">=3.9,<3.13"
 readme = "README.md"
-homepage = "https://github.com/Aleph-Alpha/aleph-alpha-client"
-packages = [{ include = "aleph_alpha_client" }]
+license = "MIT"
+dependencies = [
+    "requests>=2.28",
+    "urllib3>=1.26",
+    "aiohttp>=3.10.2",
+    "aiodns>=3.2.0",
+    "aiohttp-retry>=2.8.3",
+    "tokenizers>=0.13.2",
+    "typing-extensions>=4.5.0",
+    "Pillow>=9.2.0",
+    "tqdm>=4.62.0",
+    "python-liquid>=1.9.4",
+    "packaging>=23.2",
+]
 
-[tool.poetry.dependencies]
-python = ">=3.9,<3.13"
-requests = ">=2.28"
-urllib3 = ">=1.26"
-aiohttp = ">=3.10.2"
-aiodns = ">=3.2.0"
-aiohttp-retry = ">=2.8.3"
-tokenizers = ">=0.13.2"
-typing-extensions = ">=4.5.0"
-Pillow = ">=9.2.0"
-tqdm = ">=4.62.0"
-python-liquid = ">=1.9.4"
-packaging = ">=23.2"
-sphinx = { version = ">=6.1.3", optional = true }
-sphinx-rtd-theme = { version = ">=1.2.1", optional = true }
+[project.urls]
+Homepage = "https://github.com/Aleph-Alpha/aleph-alpha-client"
 
-[tool.poetry.group.dev.dependencies]
-pytest = "^8.3.4"
-pytest-cov = "^6.0.0"
-pytest-dotenv = "^0.5.2"
-pytest-httpserver = "^1.1.1"
-pytest-aiohttp = "^1.1.0"
-mypy = "^1.15.0"
-pre-commit = "^4.1.0"
-ruff = ">=0.9.7,<0.12.0"
-types-requests = "^2.32.0.20241016"
-types-pillow = "^10.2.0.20240822"
-types-tqdm = "^4.67.0.20241221"
-nbconvert = "^7.16.6"
-ipykernel = "^6.29.5"
+[dependency-groups]
+dev = [
+    "pytest>=8.3.4,<9",
+    "pytest-cov>=6.0.0,<7",
+    "pytest-dotenv>=0.5.2,<0.6",
+    "pytest-httpserver>=1.1.1,<2",
+    "pytest-aiohttp>=1.1.0,<2",
+    "mypy>=1.15.0,<2",
+    "pre-commit>=4.1.0,<5",
+    "ruff>=0.9.7,<0.12.0",
+    "types-requests>=2.32.0.20241016,<3",
+    "types-pillow>=10.2.0.20240822,<11",
+    "types-tqdm>=4.67.0.20241221,<5",
+    "nbconvert>=7.16.6,<8",
+    "ipykernel>=6.29.5,<7",
+]
+docs = ["sphinx>=6.1.3", "sphinx-rtd-theme>=1.2.1"]
 
-[tool.poetry.extras]
-docs = ["sphinx", "sphinx-rtd-theme"]
+[tool.hatch.build.targets.sdist]
```

Comment:
Why do we need `hatch` here?

Reply:
we could also use `setuptools`, but `hatch` is the default build backend you get when doing `uv init --lib`.